build/applicationHost.xdt

```diff
@@ -4,7 +4,7 @@
     <httpCompression>
       <scheme name="br" xdt:Locator="Match(name)" xdt:Transform="Remove" />
       <scheme name="gzip" xdt:Locator="Match(name)" xdt:Transform="Remove" />
-      <scheme name="br" dll="%XDT_EXTENSIONPATH%\%XDT_BITNESS%\iisbrotli.dll" xdt:Transform="Insert" xdt:Locator="Match(name)" />
+      <scheme name="br" dll="%XDT_EXTENSIONPATH%\%XDT_BITNESS%\iisbrotli.dll" dynamicCompressionLevel="4" staticCompressionLevel="11" xdt:Transform="Insert" xdt:Locator="Match(name)" />
```

Comment:
What is the reason for setting the level 4 and 11?

Reply:
After some reading online i found lvl 4-7 to be recommended the most for dynamic content.

For static content i expect it to be cached by IIS so thats why i put 11 there. Though I realize now that not everyone might have static file caching enabled. So i could live without it.

Reply:
@Phoenix359 Thanks. I found google's official nginx module is using default compression level 6.
https://github.com/google/ngx_brotli

I want to match this setting, what do you think?

Reply:
Well if Google uses that value as a default it is certainly a defendable value.
Next to that I can only give you my reasoning but as you mentioned it is impossible to select an 'optimal' level for every scenario. 

First I Googled to find what level others would set for Brotli dynamic compression ratio and I found that level 4-7 was  commonly choosen. I picked level 4 because:
1. It already had a significant better compression then default Azure Gzip compression
2. It would impact server performance the least (in the 4-7 level range) 
3. In the tests i did (70KB JSON over high bandwidth internet served by low end server) this setting was also the fastest. (time to last Byte received)

If it is at all possible with these kind of extensions i would love a dropdown/ARM template setting/file somewhere to be able to persistently set these levels (Gzip & Brotli dynamic and static) myself.

Reply:
@Phoenix359 OK. I understood.
The configuration of Brotli goes with this. Thank you for your contribution.